### ISSUE TITLE
`getGroupManager()` method no longer available

### ISSUE THREAD

**Issue by eccentricdevotion** [Gnat008/PerWorldInventory#96]
My plugin needs to check if worlds share inventories. I used the following code:

```
public static boolean checkWorldsCanShare(String from, String to) {
        PerWorldInventory pwi = (PerWorldInventory) Bukkit.getServer().getPluginManager().getPlugin("PerWorldInventory");
        GroupManager gm = pwi.getGroupManager();
        Group gf = gm.getGroupFromWorld(from);
        if (gf == null) {
            return true;
        } else {
            return (gf.containsWorld(to));
        }
    }
```

In 1.8.x this is no longer possible, be nice to have it back :)

**Comment by Gnat008** [comment]
Heh I didn't actually think anyone would do anything like this. I'll see what I can come up with to make stuff like that a bit easier.

**Comment by Gnat008** [comment]
@eccentricdevotion Is this the only thing you need a hook for? I think my solution will be to provide a handy "api" class to expose some of the methods, meaning you have to do a lot less work.

**Comment by eccentricdevotion** [comment]
An API would be perfect :)

**Comment by Gnat008** [comment]
This is the current state of the API:
https://github.com/Gnat008/PerWorldInventory/blob/java-1.8/src/main/java/me/gnat008/perworldinventory/api/PerWorldInventoryAPI.java
Note that it is on the new branch that uses Java 8. I do intend to move the plugin to Java 8 very shortly.

To access the API:
`PerWorldInventory pwi = (PerWorldInventory) Bukkit.getServer().getPluginManager().getPlugin("PerWorldInventory");
PerWorldInventoryAPI api = pwi.getAPI();`
You should be able to use that. It would be a good idea to do this only once, and just pass the API object around as needed.

**Comment by eccentricdevotion** [comment]
Thanks :)

**Comment by Gnat008** [comment]
Do you want a built version that you can test this with?

**Comment by eccentricdevotion** [comment]
Forked and building now

**Comment by Gnat008** [comment]
Never heard back, going to assume this is resolved. Please reopen if this is in error.

**Comment by eccentricdevotion** [comment]
Sorry, was never notified of the "Please verify" label being added. The API works perfectly, thanks for adding it.

**Comment by Gnat008** [comment]
Woo Github! lol. No problem!
